show/main.py

```diff
@@ -228,6 +228,35 @@ def alias(interfacename):
 
     click.echo(tabulate(body, header))
 
+# 'Neighbor MetaData' subcommand ("show interfaces neighbor")
+@interfaces.command()
+@click.argument('interfacename', required=False)
+def neighbor(interfacename):
+    """Show Interface Neighbor"""
```

Comment:
This docstring is used by the Click library as the help message for the command. Please make this more descriptive.